client/package.json

```diff
@@ -26,6 +26,7 @@
     "react-native-gravatar": "^1.0.2",
     "react-native-image-picker": "^0.28.1",
     "react-native-maps": "^0.24.2",
+    "react-native-modal": "^11.0.2",
```

Comment:
why an external package? React Native has a modal component inbuilt. Use that. Thank you.